controller.go

```diff
@@ -51,9 +54,16 @@ func (c *OFController) sendEchoLoop() {
 	// send echo request forever
 }
 
-func ServerLoop() {
-	serverStr := ":6633"
-	tcpAddr, err := net.ResolveTCPAddr("tcp", serverStr)
+func ServerLoop(listenPort int) {
+	var port int
+
+	if listenPort <= 0 {
+		port = DEFAULT_PORT
```

Comment:
I think it is better to  return immediately if specified number is invalid,  than using default number.
How do you feel about it?

Actually, ServerLoop function should return error and output message when problem occurs.
However, current function does not return any value, so it is future work.